File: java/get_aps.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://api.growflux.com/v1/aps")
  .method("GET", null)
  .addHeader("Accept", "application/json")
  .addHeader("Authorization", "Bearer {{ insert token here }}")
  .build();
Response response = client.newCall(request).execute();

File: c#/get_aps.cs
var client = new RestClient("https://api.growflux.com/v1/aps");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/json");
request.AddHeader("Authorization", "Bearer {{ insert token here }}");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);